### ISSUE TITLE
Determine "best" publication and dissemination channels

### ISSUE THREAD

**Issue by matthiasfromm** [OKScienceDE/Open_Science_101#42]
The initial planning is to use Github to develop and provide open science teaching/training materials since we want the materials to be reusable and adaptable. This can be easily established by Github, as initiatives as the Software Carpentry, the Data Carpentry or Mozilla Science Lab trainings have shown.

However, there were a couple of suggestions from people on the open science mailing list to also consider other platforms for provisioning (besides Github) and/or dissemination (besides the website) due to reasons of motivation/incentives for scientists and long-term preservation of the materials.

Platforms to consider/discuss:

+ [RIO Journal](http://riojournal.com/)
+ [Zenodo](https://zenodo.org/)
+ [Figshare](https://figshare.com/)

The mailing list discussion can be followed [here](https://lists.okfn.org/pipermail/open-science/2016-May/004185.html) (first mail in the thread - please read the following ones).

**Comment by aleimba** [comment]
yep, once we have actual material in here (;-)) we can easily make releases and put those on Zenodo, Figshare etc. for storage and DOI access.

**Comment by konrad** [comment]
If we have releases we can have them automatically pulled into Zenodo.
